frontend/src/sketch.js

```diff
@@ -221,6 +246,10 @@ let simulation1 = function (p) {
       return;
     let x = Math.floor(p.mouseX / cellSize);
     let y = Math.floor(p.mouseY / cellSize);
+    if(positionRadBtn.value() == "Determine hotspot position"){
+      hotspotPositionX = x;
+      hotspotPositionY = y;
+    }
```

Comment:
We need to check if the chosen spot is a building or road.
Here we can also draw the hotspot

Reply:
if the chosen spot is a building you get an alert
hotspot coordinates are drawn